app.json

```diff
@@ -27,7 +27,8 @@
         "NSLocationAlwaysUsageDescription": "Your location will only be used to determine if you are in a valid geofence."
       },
       "bundleIdentifier": "com.hoverapp.hover",
-      "buildNumber": "1.0.3"
+      "buildNumber": "1.0.3",
```

Comment:
Update version number to 1.1.0 or 1.0.4? Since we need to build and deploy to App Store/Google Play because of the new GoogleService file